src/plots/cartesian/layout_attributes.js

```diff
@@ -284,17 +284,17 @@ module.exports = {
             role: 'info',
             editType: 'calc',
             description: [
-                'Determines a pattern on the time line that generates breaks.',
+                'Determines a pattern on the time line that generates rangebreaks.',
```

Comment:
```suggestion
                'Determines a pattern on the time line that generates breaks.',
```
Not a big deal, but a few of these were the *word* breaks, not the attribute name...